src/js/components/award/financialAssistance/FinancialAssistanceContent.jsx

```diff
@@ -60,7 +60,7 @@ const FinancialAssistanceContent = ({
     };
 
     useEffect(() => {
-        if (isSubAwardIdClicked && overview.type === '05') {
+        if (isSubAwardIdClicked) {
```

Comment:
we need to only allow scroll when the award is a grant. So block grant (02), formula grant (03), project grant (04) and cooperative agreement (05) are all valid award types to scroll. Any other financial assistance such direct payment, loan or other should not scroll.

Reply:
@jonhill13 https://data-transparency-bfs.slack.com/archives/CE518GFNV/p1579797274018500?thread_ts=1579791975.014400&cid=CE518GFNV

☝️

If anything that has a subaward tab should scroll, then we shouldn't need to check the award type, we can just rely on the boolean in redux which is only truthy when the link inside the "Sub Award ID" column on the advanced search page is clicked.

In my understanding, that link will only be present -- and consequently, the redux property will only be truthy -- when an award with a subaward tab is clicked.

Reply:
True